helm_deploy/apply-for-legal-aid/templates/NOTES.txt

```diff
@@ -1,7 +1,7 @@
 1. Get the application URL by running these commands:
 {{- if .Values.ingress.enabled }}
 {{- range .Values.ingress.hosts }}
-  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ . }}{{ $.Values.ingress.path }}
+  https://{{ . }}{{ $.Values.ingress.path }}
```

Comment:
This condition was not working properly so I'm removing it.
It now assumes that it's always https.